WcaOnRails/app/models/competition.rb

```diff
@@ -702,15 +702,15 @@ def managers
   def receiving_registration_emails?(user_id)
```

Comment:
If you look at the `registrations_mailer.rb`:

https://github.com/thewca/worldcubeassociation.org/blob/e2880132491b1eeee5d4a1b04701d6f34f71e9c9/WcaOnRails/app/mailers/registrations_mailer.rb#L8-L13

you can see we use the `receive_registration_emails` attribute (database column) directly to determine email recipients.

The `receiving_registration_emails?` only determines the current setting for the given user.

If we want to change the default, it should be done on the database level via migration. Currently the default for `receive_registration_emails` is `true` (you can look it up in the `structure.sql` file).

Reply:
Thank you for the tip, this'll make it much easier